Software/Python/setup.py

```diff
@@ -12,7 +12,26 @@
     name="gopigo3",
     description="Drivers and examples for using the GoPiGo3 in Python",
     author="Dexter Industries",
-    url="http://www.dexterindustries.com/GoPiGo/",
+    url="http://www.dexterindustries.com/GoPiGo3/",
     py_modules=['gopigo3','easygopigo3','I2C_mutex'],
     #install_requires=open('requirements.txt').readlines(),
+    version='1.0.0',
+    long_description=" \
+    This python package includes two main modules for development: \n \
+    1. gopigo3 - This package is a low level module for controlling the GoPiGo3. \n \
+    2. easygopigo3 - This package is a higher-level module built on top of the gopigo3 module which is more user friendly.  \n \ ",
+    license='MIT',
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
+        # Indicate who your project is intended for
+        'Intended Audience :: Developers',
+        'Intended Audience :: Education',
+        'Topic :: Education',
+        'Framework :: Robot Framework',
+        'License :: OSI Approved :: MIT License',
+        'Programming Language :: Python :: 2.7',
```

Comment:
You were saying something about `Python 3`. 
Should we also support it?

My thoughts are that our *de facto* programming language should be `Python 3`, since `Python 2` is going to be obsolete very soon.
But in order to check and make it `Python 3` compatible, I'd have to test it and see if it's okay.
And I'm not talking about making it `Python 3`-compliant - that's a different ball game.

But since this takes time, we could make small steps instead and after we finish all these things, we can then focus on supporting `Python 3`.

What do you think?